examples/python-ros2-dataflow/random_turtle.py

```diff
@@ -42,31 +42,10 @@
                 case "INPUT":
                     match event["id"]:
                         case "direction":
-                            direction = {
-                                "linear": {
-                                    "x": event["value"][0],
-                                },
-                                "angular": {
-                                    "z": event["value"][5],
-                                },
-                            }
-
-                            twist_writer.publish(direction)
+                            twist_writer.publish(event["value"])
 
         case "external":
             pose = event.inner()[0].as_py()
 
             assert pose["x"] != 5.544445, "turtle should not be at initial x axis"
-            dora_node.send_output(
-                "turtle_pose",
-                pa.array(
-                    [
-                        pose["x"],
-                        pose["y"],
-                        pose["theta"],
-                        pose["linear_velocity"],
-                        pose["angular_velocity"],
-                    ],
-                    type=pa.float64(),
-                ),
-            )
+            dora_node.send_output("turtle_pose", event.inner())
```

Comment:
Do you have an idea how we could improve the log output when the pose is printed? Right now it is:

```
     Pose: -- is_valid: all not null -- child 0 type: float   [     8.304023   ] -- child 1 type: float   [     2.0427103   ] -- child 2 type: float   [     1.1957284   ] -- child 3 type: float   [     1.0905129   ] -- child 4 type: float   [     0.24720924   ]
```

It would be nice to use the proper field names instead of `child 0` etc.

Reply:
Yes absolutely. Will do.